Testing/SystemTests/tests/framework/ISIS_PowderPolarisTest.py

```diff
@@ -150,10 +200,12 @@ def requiredFiles(self):
     def runTest(self):
         # Gen vanadium calibration first
         setup_mantid_paths()
-        self.focus_results = run_focus_no_absorption(mode="PDF_NORM")
+        self.focus_results = run_focus_no_absorption("98533, 98534", mode="PDF", input_mode="Individual")
 
     def validate(self):
-        return validate_normalisation_focus_tests(self, "98534")
+        # Just check the files exists as functionality of the focussing tested elsewhere
+        self.validate_focus_files_exist("98533")
+        self.validate_focus_files_exist("98534")
```

Comment:
I think if it isn't already covered it would be good to have a new reference file here to check the contents of at least one of these runs!